src/sdk/pynni/nni/compression/torch/pruning/amc/amc_pruner.py

```diff
@@ -182,47 +169,42 @@ def __init__(
             channel_round=channel_round,
             output=self.output_dir
         )
-
         self.env = ChannelPruningEnv(
             self, evaluator, val_loader, checkpoint, args=self.env_args)
-
-        if self.job == 'train_export':
-            print('=> Saving logs to {}'.format(self.output_dir))
-            self.tfwriter = SummaryWriter(log_dir=self.output_dir)
-            self.text_writer = open(os.path.join(self.output_dir, 'log.txt'), 'w')
-            print('=> Output path: {}...'.format(self.output_dir))
-
-            nb_states = self.env.layer_embedding.shape[1]
-            nb_actions = 1  # just 1 action here
-
-            rmsize = rmsize * len(self.env.prunable_idx)  # for each layer
-            print('** Actual replay buffer size: {}'.format(rmsize))
-
-            self.ddpg_args = Namespace(
-                hidden1=hidden1,
-                hidden2=hidden2,
-                lr_c=lr_c,
-                lr_a=lr_a,
-                warmup=warmup,
-                discount=discount,
-                bsize=bsize,
-                rmsize=rmsize,
-                window_length=window_length,
-                tau=tau,
-                init_delta=init_delta,
-                delta_decay=delta_decay,
-                max_episode_length=max_episode_length,
-                debug=debug,
-                train_episode=train_episode,
-                epsilon=epsilon
-            )
-            self.agent = DDPG(nb_states, nb_actions, self.ddpg_args)
+        print('=> Saving logs to {}'.format(self.output_dir))
```

Comment:
suggest to replace `print` with `logging`

Reply:
Thanks, fixed.